### DISCUSSION TITLE
Keep tfly between worlds

### DISCUSSION THREAD

**Discussion by Superola86** [Zrips/CMI#2699] [Feature Requests]
The ./tfly command was made to give temp fly till the players flytime ends or the player logs out.
The player also looses the tfly if the player changes world, the time is still there but the player cant fly.

**Comment by mrfloris** [comment]
As an option, yes, I would love this, but pls do leave it off by default. We're using this for events etc, and would love to reset back to no temp fly if they leave early so they dont abuse this and fly in skyblock or survival world.

If there's a `-keep` toggle for true/false, that would be super handy.

**Comment by Sikatsu** [comment]
I would like this too. I have players that switch to the spawn world for their vote keys and already lose their temp fly. @Zrips

**Comment by Momshroom** [comment]
Hi, I see this is a pretty old thread, but I would love it if we could specify worlds.  Ie /tfly player +100 world  or /tfly slyer +100 world1, world2, world3, and as long as they're in a listed world it works, but not in others.  OR to be able to configure in the config worlds that tfly will not work in and have it not lost on world changes, just not work in those worlds.

I would love to be able to give tfly in a group of specific worlds that won't go away in the time period no matter where my players go, but won't function in other worlds at all.
ie: /tfly (player) +1000 world, world_nether,world_end and player can fly in any of those and change, but if they go to spawn or the pvp world, they can't, but when they return to one of those worlds they can fly again as long as their timer hasn't expired.
Thank you!
